features/step_definitions/purap/requisition.rb

```diff
@@ -1,6 +1,23 @@
-And /^I create a Requisition with required Delivery and Additional Institutional information populated$/ do
+And /^I create a Requisition with required Chart-Organization, Delivery and Additional Institutional information populated$/ do
   visit(MainPage).requisition
   on RequisitionPage do |req_page|
+    # Check Chart/Org associated with the user; if either one does not exist, use corresponding parameter default value and perform lookup
+    chart,org = req_page.chart_org_readonly.split('/')
+    (chart.nil? || chart.empty?) ? (nil) : (chart = chart.strip())
+    (org.nil? || org.empty?) ? (nil) : (org = org.strip())
+    if ((org.nil? || org.empty?) || (chart.nil? || chart.empty?))
+      req_page.chart_org_search
+      on OrganizationLookupPage do |org_lookup|
+        (chart.nil? || chart.empty?) ? (chart_for_lookup = get_aft_parameter_value(ParameterConstants::DEFAULT_CHART_CODE)) : (chart_for_lookup = chart)
```

Comment:
Your suggestion would make the code more readable and reduce local variables.
Refer to full code snippet below.